examples/huggingface/pytorch/code-generation/quantization/run_generation.py

```diff
@@ -28,7 +28,7 @@
     "--model", nargs="?", default="bigcode/starcoderbase", const="bigcode/starcoderbase"
 )
 parser.add_argument("--trust_remote_code", default=False)
-parser.add_argument("--revision", default="main", type=str)

```

Comment:
why changing the parameter? it will introduce incompatible change.

Reply:
_commit_hash can work, revision can used to get the correct config, but for model_class it failed. it is not incompatible change, I only add _commit_hash and doesn't delete revision.